### PR TITLE
fix: pnl report improvements

### PR DESCRIPTION
**Pull request by lukicenturi**
## Checklist

- [ ] Update wording on PnL overview
image

- [ ] Move cost basis toggle for better noticeability
image

### CODE REVIEW

File: frontend/app/src/utils/text.ts [hunk 1]
```diff
@@ -21,3 +21,104 @@ export const toCapitalCase = (string: string): string => {
...
+
+/**
+ * Returns the plural of an English word.
+ *
+ * @export
+ * @param {string} word
+ * @param {number} [amount]
+ * @returns {string}
+ */
+export function pluralize(word: string, amount?: number): string {
```
Comment (kelsos) [inline frontend/app/src/utils/text.ts:33]: is this required for some protocol?

**Review by kelsos — APPROVED** [review]
code looks good :+1:

File: frontend/app/src/utils/text.ts (hunk 1 above)
Comment (lukicenturi) [reply frontend/app/src/utils/text.ts:33]: This was asked by Left to pluralize the type 
image

Comment (LefterisJP) [reply frontend/app/src/utils/text.ts:33]: Hmm but some don't have plural :D  like staking :wink: 

The rest look fine.

Comment (lukicenturi) [reply frontend/app/src/utils/text.ts:33]: Interesting, where I can see the list of the possible types? @LefterisJP I'll try to exclude them manually

Comment (LefterisJP) [reply frontend/app/src/utils/text.ts:33]: We may add more later but these are here: https://github.com/rotki/rotki/blob/435742c3bd305a6551250984c04ddf74a66009cd/rotkehlchen/accounting/mixins/event.py#L14-L24

I think only staking would need an exception

Comment (lukicenturi) [reply frontend/app/src/utils/text.ts:33]: Done ser 
image

Comment (LefterisJP) [reply frontend/app/src/utils/text.ts:33]: Thanks

**Review by LefterisJP — APPROVED** [review]
LGTM Luki. Thanks.

**Pull request merged**